### PR TITLE
refactor: delete 8.7 importer

### PR DESCRIPTION
**Pull request by ChrisKujawa**
## Description

8.7 Importer is no longer used, and was replaced
by the Camunda Exporter

## Checklist

- [ ] for CI changes:
  - [ ] structural/foundational changes signed off by [CI DRI](https://github.com/cmur2)
  - [ ] [ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/ci.yml) modifications comply with ["Unified CI" requirements](https://github.com/camunda/camunda/wiki/CI-&-Automation#workflow-inclusion-criteria)

## Related issues

related #24441

### CODE REVIEW

File: .github/workflows/tasklist-e2e-tests.yml [hunk 1]
```diff
@@ -51,9 +51,9 @@ jobs:
         image: camunda/zeebe:8.6.0-alpha3
```
Comment (mustafadagher) [inline .github/workflows/tasklist-e2e-tests.yml:51]: ❓ @panagiotisgts should this be changed to `camunda/zeebe:SNAPSHOT`

**Comment by mustafadagher** [comment]
💭 I have a feeling that the e2e tests fail because the client/webapp uses an `8.6` version of the camunda8 nodejs sdk.

@sdorokhova do you have an idea about (how to fix) this?

**Review by mustafadagher — APPROVED** [review]

**Comment by ChrisKujawa** [comment]
> 💭 I have a feeling that the e2e tests fail because the client/webapp uses an `8.6` version of the camunda8 nodejs sdk.
> 
> @sdorokhova do you have an idea about (how to fix) this?

@pedesen do you have input on this?

**Comment by pedesen** [comment]
> 💭 I have a feeling that the e2e tests fail because the client/webapp uses an 8.6 version of the camunda8 nodejs sdk.

@mustafadagher @ChrisKujawa we're mostly using the [ZeebeGrpcClient](https://camunda.github.io/camunda-8-js-sdk/classes/index.Zeebe.ZeebeGrpcClient.html) from camunda8 nodejs sdk, which seems to be fine.

But we're also using [ZeebeRestClient](https://camunda.github.io/camunda-8-js-sdk/classes/index.Zeebe.ZeebeRestClient.html) to complete user tasks in the e2e tests, which is the reason why the processInstanceListeners tests are failing. I guess we need to find another way to achieve this in the tests? cc @vitorwtavares

**Comment by ChrisKujawa** [comment]
> > 💭 I have a feeling that the e2e tests fail because the client/webapp uses an 8.6 version of the camunda8 nodejs sdk.
> 
> @mustafadagher @ChrisKujawa we're mostly using the [ZeebeGrpcClient](https://camunda.github.io/camunda-8-js-sdk/classes/index.Zeebe.ZeebeGrpcClient.html) from camunda8 nodejs sdk, which seems to be fine.
> 
> But we're also using [ZeebeRestClient](https://camunda.github.io/camunda-8-js-sdk/classes/index.Zeebe.ZeebeRestClient.html) to complete user tasks in the e2e tests, which is the reason why the processInstanceListeners tests are failing. I guess we need to find another way to achieve this in the tests? cc @vitorwtavares

Ah I see this was replaced right? 

Could we disable these tests for now?

**Comment by mustafadagher** [comment]
> @mustafadagher @ChrisKujawa we're mostly using the [ZeebeGrpcClient](https://camunda.github.io/camunda-8-js-sdk/classes/index.Zeebe.ZeebeGrpcClient.html) from camunda8 nodejs sdk, which seems to be fine.
> 
> But we're also using [ZeebeRestClient](https://camunda.github.io/camunda-8-js-sdk/classes/index.Zeebe.ZeebeRestClient.html) to complete user tasks in the e2e tests, which is the reason why the processInstanceListeners tests are failing. I guess we need to find another way to achieve this in the tests? cc @vitorwtavares

@pedesen @vitorwtavares 
Is the issue related to the rest service front-end related? because backend-wise, the v1 rest api should be still working.

I thought it has something to do with the sdk version or such.. 

We'd appreciate any help, we want to fix and merge this ASAP.

**Comment by pedesen** [comment]
We also need to replace the importer in the local dev/e2e setup: https://github.com/camunda/camunda/blob/main/operate/Makefile

Otherwise we're not able to run Operate or e2e tests locally.

**Comment by pedesen** [comment]
> Is the issue related to the rest service front-end related? because backend-wise, the v1 rest api should be still working.

@mustafadagher Then something else might be the issue. We're looking into it.

File: operate/config/zeebe.cfg.yaml [hunk 2]
```diff
@@ -1,8 +1,7 @@
-zeebe.broker.exporters.elasticsearch:
-  className: io.camunda.zeebe.exporter.ElasticsearchExporter
+zeebe.broker.exporters.camundaexporter:
```
Comment (ChrisKujawa) [inline operate/config/zeebe.cfg.yaml:1]: 👍🏼

**Comment by mustafadagher** [comment]
Failing e2e are being investigated separately ([slack](https://camunda.slack.com/archives/C06F0GLJNFM/p1733471637652659))

**Pull request merged**